site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1036,32 +1043,39 @@ components:
    examples:
       HashedPasswordExample:
          value:
-            auth-id: sensor1
-            type: hashed-password
-            secrets: [{
-               "not-after": "2027-12-24T19:00:00Z",
-               "pwd-hash": "AQIDBAUGBwg=",
-               "salt": "Mq7wFw==",
-               "hash-function": "sha-512"
-            }]
+            [{
+                auth-id: sensor1,
+                type: hashed-password,
+                secrets: [{
+                             "not-after": "2027-12-24T19:00:00Z",
+                             "pwd-hash": "AQIDBAUGBwg=",
```

Comment:
now that we have deprecated these fields, I do not think that we should use them in example payloads, wouldn't you agree?

Reply:
I was thinking about that, but decided to leave it in for the time being as we have a lot of examples everywhere with this. I'd revisit all docs and examples as part of the #1565.

Reply:
I see